modules/porous_flow/test/tests/hysteresis/tests

```diff
@@ -0,0 +1,151 @@
+[Tests]
+  [./except01]
+    type = 'RunException'
+    input = 'except01.i'
+    expect_err = 'The Dictator proclaims that the number of fluid phases is 1 while you have foolishly entered 1 for your liquid phase number.  Remember that indexing starts at 0.  Be aware that the Dictator does not tolerate mistakes.'
+    requirement = 'If the liquid phase number exceeds the number of phases in the system, PorousFlow shall produce an appropriate error message and exit'
+    design = 'PorousFlowHysteresisOrder.md'
```

Comment:
You can put 'issues' and 'design' at the top-level if you want.

```
[Tests]
   issues = '#12345'
   design = 'Andrew.md'
```

Within each block using 'issues' or 'design' again will override what is provided in the top level.

Reply:
Thanks for that idea.  In this case i don't think i will, as there will be a zillion more tests in `modules/porous_flow/test/tests/hysteresis/tests` as i continue to develop hysteresis in PorousFlow.